src/internal/item-container/styles.scss

```diff
@@ -25,6 +25,10 @@
   z-index: 2000;
   position: fixed;
 }
+.transformed.keyboardDragged {
```

Comment:
nit: why do we define it as `.transformed.keyboardDragged`  and not just `.keyboardDragged`? That is not an issue, but is asymmetrical to the `.dragged` style above.